docker/alpine/schema.prisma

```diff
@@ -0,0 +1,14 @@
+generator client {
```

Comment:
Why 2 schemas? one in docker/alpine/schema.prisma and one in docker/alpine/prisma/schema.prisma